annotationPipeline/src/main/java/org/cbioportal/annotation/pipeline/MutationRecordReader.java

```diff
@@ -106,25 +107,31 @@ public void handleLine(String line) {
         reader.close();
         
         for(MutationRecord record : mutationRecords) {
-            AnnotatedRecord annotatedRecord = annotator.annotateRecord(record, replace, isoformOverride, true);
-            if (annotatedRecord.getHGVSc().isEmpty() && annotatedRecord.getHGVSp().isEmpty()) {
-                String message = "Failed to annotate record for sample " + record.getTumor_Sample_Barcode() + 
-                            " and variant (chr,start,end,ref,alt): (" + record.getChromosome() + "," + 
-                            record.getStart_Position() + "," + record.getEnd_Position() + "," + 
-                            record.getReference_Allele() + "," + annotatedRecord.getTumor_Seq_Allele2() + ")";                
-                if (annotator.isHgvspNullClassifications(annotatedRecord.getVariant_Classification())) {
-                    failedNullHgvspAnnotations++;
-                    message += " - Ignoring record with HGVSp null classification: " + annotatedRecord.getVariant_Classification();
-                }
-                else if (annotatedRecord.getChromosome().equals("M")) {
-                    failedMitochondrialAnnotations++;
-                    message += " - Cannot annotate mitochondrial variants at this time.";
-                }
-                failedAnnotations++;
-                LOG.info(message);
-            }            
-            annotatedRecords.add(annotatedRecord);
-            header.addAll(annotatedRecord.getHeaderWithAdditionalFields());
+            try {
```

Comment:
Why does this try need to be around all this code? Couldn't it just be around the actual call to genome nexus at line 111?